src/components/Claimer.tsx

```diff
@@ -66,6 +93,70 @@ const Claimer: React.FC<IClaimer> = ({
     );
   };
 
+  if (isRefunding) {
+    return (
+      <>
+    <Box
+        w="100%"
+        maxWidth="1200px"
+        ml="auto"
+        mr="auto"
+        mt="60px"
+        mb="60px"
+        px={['20px', '20px', 0]}
+      >
+        <Box
+          textAlign="center"
+          border="solid 1px"
+          borderRadius="5px"
+          borderColor="lid.stroke"
+          bg="white"
+          display="block"
+          w="100%"
+          mb="20px"
+          p="20px"
+        >
+          <Text fontSize={['24px', '36px']} fontWeight="bold">
+            {`Claim Refund`}
+          </Text>
+    
+          <Input
+            fontSize="18px"

```

Comment:
Not sure you just followed the design, but I think we don't need the input box for refund address.
We can just use the address connected to your dapp and in this case you can move getRefundableEth function call to the MainApp.tsx using multicalljs

Reply:
Yeah I was told to make it so they have to enter an address. Can change it to just use the connect wallet though.

Reply:
Ok then let's leave it for now